src/components/menu/Footer.tsx

```diff
@@ -15,6 +15,7 @@ class Footer extends React.Component<object, object> {
                             Based on the first version by <span className='is-family-code'>pip</span>&nbsp; &bull;
                             Contribute on <a href="https://github.com/SiegeEngineers/aoe2cm2">GitHub</a>
                         </p>
+                        <p>❤️ New Api interface <a href="/API">API</a> ❤️</p>
```

Comment:
I would put this next to the "Contribute on GitHub" link and just call it "API"

Reply:
Also, I think you want to use `<Link to="/api">API</Link>` here if I recall correctly

Reply:
i changed it to the new line but using link to messes things up

Reply:
alright then :slightly_frowning_face: